functionary/ui/templates/partials/table_template.html

```diff
@@ -0,0 +1,31 @@
+{% extends "django_tables2/bootstrap5.html" %}
+{% load render_table from django_tables2 %}
+{% load querystring from django_tables2 %}
+{% block table.thead %}
+    {% if table.show_header %}
+        <thead {{ table.attrs.thead.as_html }}>
+            <tr>
+                {% for column in table.columns %}
+                    <th {{ column.attrs.th.as_html }} scope="col">
+                        {% if column.orderable %}
+                            <a href="{% querystring table.prefixed_order_by_field=column.order_by_alias.next %}"
+                               title="Click to toggle sort"
+                               class="text-nowrap">
+                                {{ column.header }}
+                                {% if 'asc' in column.attrs.th.as_html %}
+                                    <i class="fa-solid fa-arrow-up-a-z" title="Sorted in ascending order"></i>
```

Comment:
This should be `fa-arrow-down-a-z`.  If both the arrow and the letters change direction, then the meaning of the two icons feels the same.  I figured I'd check what someone else does, with my first instinct being Excel, and sure enough:

![image](https://user-images.githubusercontent.com/86254668/231331730-0e5f321b-b282-465a-b026-e5c55bda7190.png)

Having said that, I don't personally feel like this icon is an improvement over the old one.  Seeing the AZ or ZA on columns that are not alphabetical listings (which I guess is just dates) feels odd to me, but also doesn't feel like it conveys any more that a simple up or down indicator.  If you really hate the chevron, I think `sort-up` / `sort-down` could look fine (though those are difficult to align, which is why I originally chose the chevron) as would plain old `arrow-up` / `arrow-down`